.github/workflows/operator_ci.yaml

```diff
@@ -0,0 +1,181 @@
+          file: build/Dockerfile.${{ matrix.image_type }}
+          cache-from: type=local,src=/tmp/.buildx-${{ matrix.image_type }}
+          cache-to: type=local,dest=/tmp/.buildx-new-${{ matrix.image_type }}
+        # Temp fix
+        # https://github.com/docker/build-push-action/issues/252
+        # https://github.com/moby/buildkit/issues/1896
+      - name: Move cache
+        run: |
+          [[ -G /tmp/.buildx-${{ matrix.image_type }} ]] && rm -rf /tmp/.buildx-${{ matrix.image_type }}
+          mv /tmp/.buildx-new-${{ matrix.image_type }} /tmp/.buildx-${{ matrix.image_type }}
+    outputs:
+      operator_specific_tag_for_test: "${{ needs.prepare_env.outputs.operator_specific_tag_for_test }}"
+
+  create_cluster:
+    runs-on: ubuntu-latest
+    name: "Create cluster"
+    needs: build_and_push_images
+    env:
+      csiblock_docker_registry_username: ${{ secrets.CSIBLOCK_DOCKER_REGISTRY_USERNAME }}
+      controller_repository_for_test: '${{ secrets.CSIBLOCK_DOCKER_REGISTRY_USERNAME }}/ibm-block-csi-controller'
+      node_repository_for_test: '${{ secrets.CSIBLOCK_DOCKER_REGISTRY_USERNAME }}/ibm-block-csi-node'
+      operator_image_repository_for_test: '${{ secrets.CSIBLOCK_DOCKER_REGISTRY_USERNAME }}/ibm-block-csi-operator'
+      operator_specific_tag_for_test: "${{ needs.build_and_push_images.outputs.operator_specific_tag_for_test }}"
+      operator_image_for_test: "${{ secrets.CSIBLOCK_DOCKER_REGISTRY_USERNAME }}/ibm-block-csi-operator:${{ needs.build_and_push_images.outputs.operator_specific_tag_for_test }}"
+      operator_yaml: "deploy/installer/generated/ibm-block-csi-operator.yaml"
+      cr_yaml: "config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml"
+    timeout-minutes: 7
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: Retrieve ci dependencies
+        uses: actions/download-artifact@v2
+        with:
+          name: ci_dependencies
+          path: /home/runner
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-${{ hashFiles('dev-requirements.txt') }}
+      - name: Get driver images tag from branch
+        id: driver_images_tag_from_branch
+        run: |
+          build/ci/github_actions/driver/get_driver_images_tag_from_branch.sh
+        env:
+          github_token: ${{ secrets.CSIBLOCK_GITHUB_TOKEN }}
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.1.0
+        with:
+          version: v0.11.1
+      - name: Wait for k8s cluster to be ready
+        run: |
+          build/ci/github_actions/wait_for_k8s_ready.sh
+      - name: Deploy ibm block csi operator
+        run: |
+          build/ci/github_actions/operator/deploy_operator.sh
+      - name: Deploy ibm block csi driver
```

Comment:
why is this under [`create_cluster`](https://github.com/IBM/ibm-block-csi-operator/blob/1ed494f3c9887bdc16dbf8c6470bf21d440eb1bf/.github/workflows/operator_ci.yaml#L118)?